src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
-export interface Figure {
+type Figures = 'triangle' | 'circle' | 'rectangle';
```

Comment:
Better to use `enum` for these values, cause you use them in your code as values